cmake/inputs/TileDB-packages.cmake.in

```diff
@@ -0,0 +1,113 @@
+#
+# FindTileDB_EP.cmake
+#
+#
+# The MIT License
+#
+# Copyright (c) 2023 TileDB, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+function(DOWNLOAD_TILEDB FROM_SOURCE)
+    # Available builds
+    set(URL_TILEDB_LINUX_X86_64 "{{URL_TILEDB_LINUX_X86_64}}")
+    set(SHA_TILEDB_LINUX_X86_64 "{{SHA_TILEDB_LINUX_X86_64}}")
+
+    set(URL_TILEDB_LINUX_X86_64_NOAVX2 "{{URL_TILEDB_LINUX_X86_64_NOAVX2}}")
+    set(SHA_TILEDB_LINUX_X86_64_NOAVX2 "{{SHA_TILEDB_LINUX_X86_64_NOAVX2}}")
+
+    set(URL_TILEDB_MACOS_ARM64 "{{URL_TILEDB_MACOS_ARM64}}")
+    set(SHA_TILEDB_MACOS_ARM64 "{{SHA_TILEDB_MACOS_ARM64}}")
+
+    set(URL_TILEDB_MACOS_X86_64 "{{URL_TILEDB_MACOS_X86_64}}")
+    set(SHA_TILEDB_MACOS_X86_64 "{{SHA_TILEDB_MACOS_X86_64}}")
+
+    set(URL_TILEDB_WINDOWS_X86_64 "{{URL_TILEDB_WINDOWS_X86_64}}")
+    set(SHA_TILEDB_WINDOWS_X86_64 "{{SHA_TILEDB_WINDOWS_X86_64}}")
+
+    # Source build
+    set(URL_TILEDB_SOURCE "{{URL_TILEDB_SOURCE}}")
+    set(SHA_TILEDB_SOURCE "{{SHA_TILEDB_SROUCE}}")
+
+    if(${FROM_SOURCE})
+        ExternalProject_Add(ep_tiledb
+                PREFIX "externals"
+                URL "${URL_TILEDB_SOURCE}"
+                URL_HASH SHA1=${SHA_TILEDB_SOURCE}
```

Comment:
SHA-1 is insecure; we should be using at least SHA-256.

Reply:
I agree, I would like to change it if everyone is OK with it